integration-test-common/src/test/java/org/apache/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -52,6 +52,9 @@ public class ContainerSuite implements Closeable {
 
   private static Network network = null;
   private static volatile HiveContainer hiveContainer;
+
+  // Enable the Ranger plugin in the Hive container
+  private static volatile HiveContainer hiveRangerContainer;
```

Comment:
The name should be `RangerHiveContainer`? the name `hiveRangerContainer` make me guess it's a ranger container NOT a hive container.

Reply:
RangerHiveContainer or HiveRangerContainer can be ambiguous, the comments make that clear.